scripts/upload-file-to-s3.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""
+Usage:
+    scripts/upload-file-to-s3.py
+        <stage> <file_path> <bucket_category> [<remote_path>] [<remote_name>] [--dry-run] [--public]
+
+Options:
+    -h --help   Show this screen.
+    --stage=<stage>  The stage being updated
+    --file_path=<file_path>  Path of file to be saved in S3
+    --bucket_category=<bucket_category>  Bucket category, used to get name ie agreements/ communications/ documents
+    --remote_path=<remote_path>  Path to save the file at inside the bucket
+    --download_name=<download_name>  Suggested name for a browser to download, part of Content-Disposition header
+    --dry-run  Don't actually perform the upload
+    --public  Trigger the public-read acl option, defaults to private
+"""
+import os
+import sys
+sys.path.insert(0, '.')
+
+from docopt import docopt
+
+from dmutils.s3 import S3
+
+from dmscripts.bulk_upload_documents import get_bucket_name
+
+
+if __name__ == '__main__':
+    arguments = docopt(__doc__)
+
+    # Get arguments
+    stage = arguments['<stage>']
+    file_path = arguments['<file_path>']
+    bucket_category = arguments['<bucket_category>']
+    remote_path = arguments.get('<remote_path>', '')
+    download_name = arguments.get('<remote_name>', '')
+    dry_run = arguments['--dry-run']
+    public = arguments['--public']
+
+    # Set defaults
+    acl = 'public-read' if public else 'private'
+    download_name = download_name if download_name else os.path.basename(file_path)
+
+    # Grab bucket
+    bucket = S3(get_bucket_name(stage, bucket_category))
```

Comment:
Instead of giving a bucket category, is there value in just supplying a bucket name? Would mean the script is useable if we want to upload to a bucket outside of that pattern at any point.